### PR TITLE
Search mobile layout

### PR DESCRIPTION
**Pull request by ger-benjamin**
Layout for search's results and the 'clear search' button

### CODE REVIEW

File: contribs/gmf/examples/style/mobileapp-search.less [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      font-size: 0.8em;
+    }
+  }
+}
+
+//For tablet only
+@media (min-width: @screen-sm-min) {
+  .tt-menu {
+    width: 33vw;
+    border-top-right-radius: 20px;
```
Comment (pgiraud) [inline contribs/gmf/examples/style/mobileapp-search.less]: Is there any reason for this radius on the top right border? Cosmetic ?
Personaly I don't like it.  ;-)

screenshot from 2015-11-17 16 06 13

File: contribs/gmf/examples/style/mobileapp-search.less [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  left: 0;
+  right: 0;
+  top: 50px!important;
+  bottom: 0;
+  overflow-x: hidden;
+  overflow-y: auto;
+  width: 100vw;
+  background-color: white;
+  border: solid 1px black;
+  border-top: none;
```
Comment (pgiraud) [inline contribs/gmf/examples/style/mobileapp-search.less]: I'd prefer a border all around the tt-menu element and border-bottom for each item in the list.

File: contribs/gmf/src/directives/search.js [hunk 3]
```diff
@@ -105,6 +118,14 @@ gmf.SearchController = function($scope, $compile,
   this.map_ = map;
 
   /**
+   * Use or not a clear button to clear the search's dropdown.
```
Comment (pgiraud) [inline contribs/gmf/src/directives/search.js]: This doesn't really clears the dropdown. This is a consequence of clearing the text input. I'd use the following:
"Whether or not to show a button to clear the search text."

File: externs/ngeox.js [hunk 4]
```diff
@@ -327,13 +327,13 @@ ngeox.MeasureEvent.prototype.feature;
 
 /**
  * @typedef {{
- *   opened: (function()|undefined),
- *   closed: (function()|undefined),
- *   cursorchanged: (function(jQuery.Event, Object,
+ *   opene: (function()|undefined),
```
Comment (pgiraud) [inline externs/ngeox.js]: typo?

File: externs/ngeox.js [hunk 5]
```diff
@@ -343,31 +343,31 @@ ngeox.SearchDirectiveListeners;
...
  */
-ngeox.SearchDirectiveListeners.prototype.cursorchanged;
+ngeox.SearchDirectiveListeners.prototype.cursorchange;
 
 
 /**
  * @type {function(jQuery.Event, Object, TypeaheadDataset)|undefined}
  */
-ngeox.SearchDirectiveListeners.prototype.selected;
+ngeox.SearchDirectiveListeners.prototype.selecte;
```
Comment (pgiraud) [inline externs/ngeox.js]: typo?

**Comment by pgiraud** [comment]
Can you please rebase your work onto master? It looks like the commit for the close button is not required anymore.

**Comment by ger-benjamin** [comment]
Yep, I'll correct and rebase this PR today.

**Comment by ger-benjamin** [comment]
@pgiraud ~Rebased and corrected

**Comment by pgiraud** [comment]
OK. Please go ahead and merge this.

**Pull request merged**